clusters/app.ci/ci-cluster-pool/ci-ocp-4-7-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4-7-0-amd64-aws-us-east-1
+  namespace: ci-cluster-pool
+  labels:
+    product: ocp
+    version: 4.7.0
```

Comment:
ftr, I think the z stream should not be in the label. Ppl don't care about that and its annoying to bump it every time a new pach release is published

Reply:
Updated.